recipes/fits2image/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "fits2image" %}
+{% set version = "0.4.9" %}
+{% set python_min = "3.8" %}
```

Comment:
Conda-forge doesn't support EOL Python. If this is really necessary you can keep it, but don't expect any support in case of failures/problems.

```suggestion
```